src/Application.vala

```diff
@@ -61,10 +63,17 @@ namespace Maya {
         }
 
         protected override void activate () {
+            new Calendar.TodayEventMonitor ();
```

Comment:
Should this be done in `activate`? it looks like Mail does the equivalent in `startup`

Reply:
Well, Calendar doesn't have a `startup` method. This work since TodayEventMonitor is now marked as a `[SingleInstance]` class and the construct method will only be called one time.

Reply:
Well, Calendar doesn't have a `startup` method. This work since TodayEventMonitor is marked as a `[SingleInstance]` class and the construct method will only be called one time.